ursa/graph/graph.py

```diff
@@ -1,22 +1,32 @@
 import ray
 from .vertex import _Vertex
 from .vertex import _DeletedVertex
+from .utils import write_vertex
 
 
 @ray.remote(num_cpus=2)
 class Graph(object):
     """This object contains reference and connection information for a graph.
 
     @field vertices: The dictionary of _Vertex objects.
+    @field versions_to_store: The number of versions to keep in memory for each
+                              node
+    @field graph_id: The globally unique ID to identify this graph.
```

Comment:
Does the graph really need to know its own ID?

Reply:
I did this because the filepath uses the graph_id to determine a unique file path for each graph. The filepath is currently given by `graph_id/vertex.key/transaction_id.dat`. I can change this if needed.

Reply:
Can `local_manager` just pass it as a parameter? Same with `versions_to_store`.

Reply:
Done.